docs/bom.adoc

```diff
@@ -0,0 +1,210 @@
+== Arquillian Cube BOM - Unified Dependecies
+
+This aims to fulfill requirements of unify naming & versions.
+
+=== Usage
+
+Include the following snippet in your pom.xml file:
+
+[source, xml]
+.pom.xml
+----
+<properties>
+    <version.arquillian_cube>${latest_released_version}</version.arquillian_cube>
+</properties>
+
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>org.arquillian.cube</groupId>
+            <artifactId>arquillian-cube-bom</artifactId>
+            <version>${version.arquillian_cube}</version>
+            <scope>import</scope>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+----
+
+Then include the individual modules as you see fit, by simply depending on the unified pom name:
+
+[source, xml]
+----
+<dependencies>
+    <dependency>
+        <groupId>org.arquillian.cube</groupId>
+        <artifactId>arquillian-cube-docker</artifactId>
+        <scope>test</scope>
+        <type>pom</type>
+    </dependency>
+</dependencies>
+----
+
+=== Availbale Modules:
+
+===== Arquillian Cube API
+
+[source, xml]
+----
+    <dependency>
+        <groupId>org.arquillian.cube</groupId>
+        <artifactId>arquillian-cube-api</artifactId>
+        <scope>test</scope>
+        <type>pom</type>
+    </dependency>
+----
+
+==== Arquillain Cube SPI
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.arquillian.cube</groupId>
+    <artifactId>arquillian-cube-spi</artifactId>
+    <scope>test</scope>
+    <type>pom</type>
+</dependency>
+----
+
+==== Arquillina Cube Core
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.arquillian.cube</groupId>
+    <artifactId>arquillian-cube-core</artifactId>
+    <scope>test</scope>
+    <type>pom</type>
+</dependency>
+----
+
+==== Arquillina Cube Containerless
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.arquillian.cube</groupId>
+    <artifactId>arquillian-cube-containerless</artifactId>
+    <scope>test</scope>
+    <type>pom</type>
+</dependency>
+----
+
+==== Arquillina Cube Requirement
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.arquillian.cube</groupId>
+    <artifactId>arquillian-cube-requirement</artifactId>
+    <scope>test</scope>
+    <type>pom</type>
+</dependency>
+----
+
+==== Arquillina Cube docker
```

Comment:
And crazy idea - how about autogenerating these sections based on the content of POM? :)